inntekt/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/inntekt/HentInntektRiver.kt

```diff
@@ -54,6 +57,7 @@ class HentInntektRiver(
                 behovType = Key.BEHOV.krev(BehovType.HENT_INNTEKT, BehovType.serializer(), json),
                 transaksjonId = Key.KONTEKST_ID.les(UuidSerializer, json),
                 data = data,
+                svarKafkaKey = Key.SVAR_KAFKA_KEY.lesOrNull(KafkaKey.serializer(), data),
```

Comment:
Er det sånn at vi i først neste omgang/PR faktisk skal publisere meldinger med denne nøkkelen, og at dette bare er foreberedelsene?

Jeg forsøkte å grave meg litt ned i riveren og kom til den linja her: `?.also { context.publish(null, it) }` i `OpenRiver.kt`. Er det der vi etter hvert skal sende inn denne nøkkelen?

Reply:
Det stemmer at dette er forberedelser. `null` i `context.publish(null, it)` må byttes ut med en verdi. For alle som får `svarKafkaKey`, så skal vi bruke den verdien. For resterende så er ikke løsningen helt klar.